app/src/main/res/values/strings_questionnaire.xml

```diff
@@ -11,13 +11,96 @@
         <item>60~</item>
         <item>secret</item>
     </string-array>
-    <string-array name="questionnaire_values_age">
-        <item>10代</item>
-        <item>20代</item>
-        <item>30代</item>
-        <item>40代</item>
-        <item>50代</item>
-        <item>60代</item>
-        <item>答えたくない</item>
+    <string name="questionnaire_android_user">How long have you been using Android?</string>
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_android_user" is not translated in "ja" (Japanese) [MissingTranslation]